phpunit/wp-theme-json.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Tests theme.json related public APIs.
+ *
+ * @package Gutenberg
+ */
+
+class WP_Theme_Json_Test extends WP_UnitTestCase {
```

Comment:
These tests are not picked up:

- `npm run test:unit:php`: does not report any error (all these tests should fail)
- `npm run test:unit:php -- --group theme_json`: does not run any test (it should run one, the last of these)

Reply:
Try `npm run test:unit:php -- --filter WP_Theme_Json_Test`

Reply:
The filenames should have a `-test.php` suffix.

Reply:
Adding the suffix, fixes it. Thanks!

PS: it looks like others with the same issue were committed, fix at https://github.com/WordPress/gutenberg/pull/45288